.gitignore

```diff
@@ -1,4 +1,14 @@
 dist/
 node_modules/
-.vscode/
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+
+# Created by https://www.gitignore.io/api/visualstudiocode
+
+### VisualStudioCode ###
+.vscode/*
+!.vscode/settings.json
```

Comment:
I'd put a vote in for editorconfig, works pretty seamlessly in things that I've worked on - was meaning to put it in here but didn't get around to it